src/ModifyTodos.js

```diff
@@ -0,0 +1,56 @@
+let todos = JSON.parse(localStorage.getItem('todos')) || [];// eslint-disable-line
+
+const listGroup = document.querySelector('.todo-list-group');
+const newTask = document.querySelector('.todo-add').querySelector('input');
+const getTodos = () => {
+  const todoElement = todos.map((item) => `
+        <li class="todo-list todo-item" id=${item.index}>
+          ${item.completed === true ? `
+            <i class="fa-solid fa-check checked-icon"></i>`
+    : '<i class="fa-solid fa-square unchecked-icon"></i>'}
+          <input type="text" class=${item.completed === true ? 'decoration edit-todo' : ' edit-todo'}  value="${item.description}">
+          <span class="edit-focus-element"></span>
+          <i class="fa-solid fa-trash-can delete-icon"></i>
+          <i class="fa-solid fa-ellipsis-vertical more-icon"></i>
+        </li>`).join('');
+  listGroup.innerHTML = todoElement;
+  return listGroup;
+};
+
+const addTodos = (event) => {
+  if (newTask.value === '') return;
+  if (event.key === 'Enter' || event === 'clicked') {
+    const newTodo = {
+      description: newTask.value,
+      completed: false,
+      index: todos.length,
+    };
+    todos = [...todos, newTodo];
+    localStorage.setItem('todos', JSON.stringify(todos));
+    getTodos();
+  }
+};
```

Comment:
- [ ] It would be better if the index of the tasks is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). Kindly fix this minor issue.

```suggestion
const addTodos = (event) => {
  if (newTask.value === '') return;
  if (event.key === 'Enter' || event === 'clicked') {
    const newTodo = {
      description: newTask.value,
      completed: false,
      index: todos.length +1,
    };
    todos = [...todos, newTodo];
    localStorage.setItem('todos', JSON.stringify(todos));
    getTodos();
  }
};
```